modules/raft/src/main/java/org/apache/ignite/internal/raft/Loza.java

```diff
@@ -175,6 +170,11 @@ public void actionRequestInterceptor(ActionRequestInterceptor actionRequestInter
     /** {@inheritDoc} */
     @Override
     public CompletableFuture<Void> start() {
+        executor = new ScheduledThreadPoolExecutor(
```

Comment:
All over the codebase we create executors in constructors, why have you moved this executor to the start method?